build/dsc.rake

```diff
@@ -80,6 +80,8 @@ eod
         Rake::Task['dsc:resources:checkout'].invoke(
           official_dsc_resources_root, update_versions, composite_resources)
 
+        FileUtils.mv("#{dsc_resources_path_tmp}/xDscResources/xSQLServer", "#{dsc_resources_path_tmp}/xDscResources/SqlServerDsc", :verbose => true, :force => true)
```

Comment:
Is there another spot this could go? Seems like it at least belongs in a parameterized list and with some sort of comment about why it's necessary.

Reply:
You mean like i noted in the PR description in the TODO?